content/docs/neon-auth/claim-project.md

```diff
@@ -3,9 +3,23 @@ title: Claiming a Neon Auth project
 enableTableOfContents: true
 tag: beta
 updatedOn: '2025-05-16T19:06:06.840Z'
+redirectFrom:
+  - /docs/guides/neon-auth-claim-project
 ---
 
-Neon Auth is powered by Stack Auth under the hood. By default, Neon manages your authentication for you, so you never need to interact with Stack Auth directly. If you want to "eject" and manage your authentication project directly in Stack Auth (for advanced configuration or direct provider control), you can claim ownership of the project.
+Neon Auth is powered by Stack Auth under the hood. By default, Neon manages your authentication for you, so you do not typically need to interact with Stack Auth directly. However, there are cases where you may want to take direct control of your authentication project in the Stack Auth dashboard.
+
+## Why claim a project?
+
+Most Neon Auth features can be built using the SDKs, without claiming your project.
+
+Right now, you need to claim your project if you want to:
+
+- Add or manage OAuth providers (register client IDs/secrets, set callback URLs)
+- Whitelist domains or configure allowed callback URLs
```

Comment:
There's a `Domains` section in Neon Auth configuration, so it appears 'whitelist domains' is the one config we can now do from the Neon Auth console